airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -285,16 +284,36 @@ def request_body_json(self, stream_slice: Mapping = None, **kwargs) -> Optional[
 
 
 class SourceSlack(AbstractSource):
+    def _get_authenticator(self, config: Mapping[str, Any]):
```

Comment:
this should be backwards compatible with the previous config format to prevent a breaking change for current users

Reply:
Added backward compatibility, but for this it was necessary to remove `credentials `from `required`.
@avida described this issue in more detail in [this ](https://github.com/airbytehq/airbyte/pull/6833#issuecomment-935801375)comment.